src/components/AeCoinPanel.vue

```diff
@@ -53,6 +53,17 @@
             <price-label :price="totalTokenSupply"/>
           </td>
         </tr>
+        <tr class="ae-coin-panel__row">
+          <th class="ae-coin-panel__table-header">
+            <hint-tooltip>
+              {{ aeCoinHints.activeAccounts }}
+            </hint-tooltip>
+            Latest Active Accounts
```

Comment:
Instead of displaying "Latest Active Accounts" in the Ae coin page I would add:
- Active accounts (24h)
- Total accounts

As stats panel in the /accounts page, like it's done for /transactions, as it's more related to accounts rather than the native coin itself and seems more consistent to me?

Reply:
Yea, sure good suggestion. I can extend i.

Reply:
> displaying "Latest Active Accounts" in the Ae coin page

This was a direct requirement before tho, it think from Nikola. So I will keep it + extend it.

Reply:
Done

Reply:
Aha, got it. I mixed 2 things
I removed the Latest Active Accounts from Coin page
I requested MDW for the token holders. Its not available yet.  https://github.com/aeternity/aescan/issues/976#issuecomment-2447010788